packages/grafana-ui/src/components/Forms/FieldSet.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { FC, HTMLProps } from 'react';
+import { css, cx } from 'emotion';
+import { GrafanaTheme } from '@grafana/data';
+import { stylesFactory, useTheme } from '../../themes';
+import { Legend } from './Legend';
+
+export interface Props extends HTMLProps<HTMLFieldSetElement> {
+  children: React.ReactNode[] | React.ReactNode;
+  /** Text for the fieldset's legend */
+  label?: string;
+}
+
+export const FieldSet: FC<Props> = ({ label, children, className, ...rest }) => {
+  const theme = useTheme();
+  const styles = getStyles(theme);
+
+  return (
+    <fieldset className={cx(styles.wrapper, className)} {...rest}>
+      <>
+        {label && <Legend>{label}</Legend>}
+        {children}
+      </>
+    </fieldset>
+  );
+};
+
+const getStyles = stylesFactory((theme: GrafanaTheme) => {
+  return {
+    wrapper: css`
+      margin-bottom: ${theme.spacing.xl};
```

Comment:
Can we use `theme.spacing.formSpacingBase` for consistency?  We use it like this in i.e. `Field` component: `margin-bottom: ${theme.spacing.formSpacingBase * 2}px;`